Mathlib/Logic/Relation.lean

```diff
@@ -310,6 +310,15 @@ theorem head (hab : r a b) (hbc : ReflTransGen r b c) : ReflTransGen r a c := by
   | refl => exact refl.tail hab
   | tail _ hcd hac => exact hac.tail hcd
 
+instance : Trans r r (ReflTransGen r) where
```

Comment:
This isntance is the bad one, since the last argument of `Trans` is an `outParam`, meaning it has to be uniquely determined by the other arguments. The ones below look safe.